cmd/tsbs_generate_queries/databases/siridb/devops.go

```diff
@@ -0,0 +1,175 @@
+package siridb
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/devops"
+	"github.com/timescale/tsbs/query"
+)
+
+// Devops produces SiriDB-specific queries for all the devops query types.
+type Devops struct {
+	*devops.Core
+}
+
+// NewDevops makes an Devops object ready to generate Queries.
+func NewDevops(start, end time.Time, scale int) *Devops {
+	return &Devops{devops.NewCore(start, end, scale)}
+}
+
+// GenerateEmptyQuery returns an empty query.SiriDB
+func (d *Devops) GenerateEmptyQuery() query.Query {
+	return query.NewSiriDB()
+}
+
+func (d *Devops) getHostWhereWithHostnames(hostnames []string) string {
+	hostnameClauses := []string{}
+	for _, s := range hostnames {
+		hostnameClauses = append(hostnameClauses, fmt.Sprintf("`%s`", s))
+	}
+	combinedHostnameClause := strings.Join(hostnameClauses, "|")
+	return "(" + combinedHostnameClause + ")"
+}
+
+func (d *Devops) getHostWhereString(nhosts int) string {
+	hostnames := d.GetRandomHosts(nhosts)
+	return d.getHostWhereWithHostnames(hostnames)
+}
+
+func (d *Devops) getMetricWhereString(metrics []string) string {
+	metricsClauses := []string{}
+	for _, s := range metrics {
+		metricsClauses = append(metricsClauses, fmt.Sprintf("`%s`", s))
+	}
+	combinedMetricsClause := strings.Join(metricsClauses, "|")
+	return "(" + combinedMetricsClause + ")"
+}
+
+const goTimeFmt = "2006-01-02 15:04:05Z"
+
+// GroupByTime selects the MAX for numMetrics metrics in the group `cpu`,
+// per minute for nhosts hosts,
+// e.g. in psuedo-SQL:
```

Comment:
I just fixed this in other files, but `pseudo` is spelled wrong here. Sorry :(

Reply:
Oh fixed it:)